cheddar/cheddar-integration-aws/src/test/java/com/clicktravel/infrastructure/persistence/aws/dynamodb/DynamoDbTemplateTest.java

```diff
@@ -137,6 +135,7 @@ public void shouldFetch_withAttributeQueryOnPrimaryKey() throws Exception {
         final AttributeQuery query = mock(AttributeQuery.class);
         final Condition mockCondition = mock(Condition.class);
         when(mockCondition.getComparisonOperator()).thenReturn(Operators.EQUALS);
+        when(mockCondition.containsNonNullOrEmptyValues()).thenReturn(true);
```

Comment:
This method determines if the condition contains at least one non-null and non-empty value to compare again.  As querying Dynamo with only null or empty values causes an exception, a check is made at the start of the query method to make sure that we have a non-null and non-empty value.  If that is not the case, the method returns an empty result list without hitting Dynamo.

In this test we are testing that we invoke Dynamo, hence we want this method to say the `Condition` contains at least one non null and non empty value.